.eslintrc.js

```diff
@@ -5,8 +5,12 @@ module.exports = {
     rules: {
         "prettier/prettier": "error",
         "arrow-body-style": ["error", "as-needed"],
-        complexity: ["error", 10],
+        complexity: ["error", 20],
+        "no-param-reassign": "off",
```

Comment:
I would consider fixing these issues rather than disabling this, I think its a valid rule and exists in many of our existing projects

Reply:
Other changes look good 👍

Reply:
Cool. I will fix the `no-param-reassign` eslint.

Reply:
Fixed.